main.go

```diff
@@ -66,6 +67,11 @@ with the '--output' Flag.
 `
 
 var ignoreNoRepo bool = false
+var showVersion bool = false
```

Comment:
Codacy found an issue: [should omit type bool from declaration of var showVersion; it will be inferred from the right-hand side](https://app.codacy.com/manual/l0nax/helm-whatup/pullRequest?prid=4860442)